pkg/console/subresource/deployment/deployment.go

```diff
@@ -233,15 +234,19 @@ func withConsoleVolumes(
 	authServerCAConfigMap *corev1.ConfigMap,
 	trustedCAConfigMap *corev1.ConfigMap,
 	sessionSecret *corev1.Secret,
-	canMountCustomLogo bool) {
+	customLogos []configmapsub.CustomLogoRef) {
 	volumeConfig := defaultVolumeConfig()
 
 	caBundle, caBundleExists := trustedCAConfigMap.Data["ca-bundle.crt"]
 	if caBundleExists && caBundle != "" {
 		volumeConfig = append(volumeConfig, trustedCAVolume())
 	}
-	if canMountCustomLogo {
-		volumeConfig = append(volumeConfig, customLogoVolume())
+
+	// Update to handle multiple custom logos
+	if len(customLogos) > 0 {
```

Comment:
Update this to handle the case where multiple logos are defined in a single configmap